kudos-board-api/controllers/cardController.js

```diff
@@ -7,9 +7,10 @@ exports.getCards = async (req, res) => {
   res.json(cards);
 };
 
-// add card (by id)
+// add card (by id) Make more descriptive "getCardById"
 exports.getCard = async (req, res) => {
   try {
+    // This is inside vs outside try catch in other functions
     const id = Number(req.params.id);
```

Comment:
I've noticed being done both inside and outside the try, catch. Be consistent where you do it.